### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by schwarherz**
Update the Readme with the info from the old Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+===============================================================================
+1) First find your installation directory. This will depend on your OS, and may not be (especially in the case of Windows) the directory where you actually installed CK2 itself:
+Windows: ~\My Documents\Paradox Interactive\Crusader Kings II\mod\
+Linux: ~/.paradoxinteractive/Crusader Kings II/mod/
+Mac: ~/Documents/Paradox Interactive/Crusader Kings II/mod/
+2) If the "mod" subdirectory doesn't exist in the above location, create it.
+
+3) If you already have a "mod" sub-directory there, excellent. Make sure you delete anything related to CK2Plus prior to the manual installation -- this includes any old versions of the mod, anything that was installed by Steam Workshop (which we no longer use), etc. You want a clean slate when installing, even if you're just updating to a new mod version.
+
+4) Make sure you've downloaded the .zip file from the link at the top of this page.
```
Comment (questionmarkexclamationpoint) [inline README.md]:
```suggestion
4) Make sure you've downloaded the [latest .zip from the releases page](github.com/ck2plus/CK2Plus_Rewrite/releases/latest)
```

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+4) Make sure you've downloaded the [latest .zip from the releases page](github.com/ck2plus/CK2Plus_Rewrite/releases/latest)
+
+5) To extract the files from within, you'll first need a program like WinRar or 7-Zip. Once you have that, you should be able to just click to open the .zip file and you can move the files over to the "mod" sub-directory. Alternatively, you can use the extractor program to select the .zip file and select where you want the files extracted (the installation directory in #1).
+
+6) Start the CK2 launcher. Check the "CK2Plus" box in the list of mods. When the title screen comes up make sure the checksum (the four-letter code to the left of the title scree buttons) matches the one listed for that CK2Plus version. If so, you're good to go. If not, you've either made some other alterations, installed some other mods, or made a mistake.
```
Comment (questionmarkexclamationpoint) [inline README.md]:
```suggestion
6) Start the CK2 launcher. Check the "CK2Plus" box in the list of mods. When the title screen comes up make sure the checksum (the four-letter code to the left of the title scree buttons) matches the one listed for that CK2Plus version. If so, you're good to go. If not, you've either made some other alterations, installed some other mods, or made a mistake.
```
I'm not sure if we want to go into detail here on the breakdown between the two modules?

File: README.md [hunk 3]
```diff
@@ -1 +1,25 @@
...
+
+Useful Links
+===============================================================================
+CK2 Plus Official Forum: https://forum.paradoxplaza.com/forum/index.php?forums/ck2-plus-mod.841/
```
Comment (questionmarkexclamationpoint) [inline README.md]:
```suggestion
[CK2 Plus Official Forum](https://forum.paradoxplaza.com/forum/index.php?forums/ck2-plus-mod.841/)
```

File: README.md (hunk 2 above)
Comment (schwarherz) [reply README.md]: > I'm not sure if we want to go into detail here on the breakdown between the two modules?

We can always edit it again later to include that detail.

Comment (questionmarkexclamationpoint) [reply README.md]: Ah yikes I wrote something good here... Lemme fix this.

**Comment by questionmarkexclamationpoint** [comment]
LGTM!

**Comment by questionmarkexclamationpoint** [comment]
Thanks for taking my suggestions! 👍

**Comment by schwarherz** [comment]
No problem at all!

**Pull request merged**